addons/binding/org.openhab.binding.honeywellwifithermostat/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="honeywellwifithermostat"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="honeythermo">
+        <label>Honeywell WiFi Thermostat</label>
+        <description>Control your Honeywell WiFi thermostat over the internet with OpenHAB.</description>
+
+        <channels>
+            <channel id="sysMode" typeId="systemMode" />
+            <channel id="currentTemp" typeId="currentTemperature"/>
+            <channel id="heatSP" typeId="heatSetPoint" />
+            <channel id="coolSP" typeId="coolSetPoint" />
+            <channel id="fanMode" typeId="FanMode" />
+        </channels>
+        
+        <config-description>
```

Comment:
I followed the indentation that existed in the skeleton files generated by
the script. Is the tab spacing off? If there is a problem I can fix it.

On Wed, Aug 19, 2015 at 3:25 AM, Markus Rathgeb <notifications@github.com>
wrote:

> In
> addons/binding/org.openhab.binding.honeywellwifithermostat/ESH-INF/thing/thing-types.xml
> <https://github.com/openhab/openhab2/pull/339#discussion_r37386140>:
>
> > +        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
> > +        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
> > +
> > +    <thing-type id="honeythermo">
> > +        <label>Honeywell WiFi Thermostat</label>
> > +        <description>Control your Honeywell WiFi thermostat over the internet with OpenHAB.</description>
> > +
> > +        <channels>
> > +            <channel id="sysMode" typeId="systemMode" />
> > +            <channel id="currentTemp" typeId="currentTemperature"/>
> > +            <channel id="heatSP" typeId="heatSetPoint" />
> > +            <channel id="coolSP" typeId="coolSetPoint" />
> > +            <channel id="fanMode" typeId="FanMode" />
> > +        </channels>
> > +
> > +        <config-description>
>
> wrong indentation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab2/pull/339/files#r37386140>.
>